### PR TITLE
Hotfix: Unit test file structure, pytest back to travis

### PR DESCRIPTION
**Pull request by btseytlin**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ In general, we follow the "fork-and-pull" Git workflow.
 4. Push your local branch to your fork
 5. Submit a Pull request so that we can review your changes
 6. Write a commit message
-7. Make sure that the CI tests are GREEN
+7. Make sure that the unit tests are GREEN
```
Comment (George3d6) [inline CONTRIBUTING.md:23]: Unit tests are part of the CI tests so this need not be added

File: mindsdb/libs/helpers/general_helpers.py [hunk 2]
```diff
@@ -132,8 +133,8 @@ def get_value_bucket(value, buckets, col_stats, hmd=None):
 
     elif col_stats['data_subtype'] in (DATA_SUBTYPES.BINARY, DATA_SUBTYPES.INT, DATA_SUBTYPES.FLOAT):
         bucket = closest(buckets, value)
-    elif col_stats['data_subtype'] in (DATA_SUBTYPES.IMAGE) and hmd is not None:
-        bucket = hmd['bucketing_algorithms'][col_name].predict(np.array(imagehash.phash(Image.open(value)).reshape(1, -1)))[0]
+    # elif col_stats['data_subtype'] in (DATA_SUBTYPES.IMAGE) and hmd is not None:
```
Comment (George3d6) [inline mindsdb/libs/helpers/general_helpers.py:136]: Why is this commented out ?

File: mindsdb/libs/phases/data_analyzer/data_analyzer.py [hunk 3]
```diff
@@ -175,13 +176,51 @@ def compute_entropy_biased_buckets(hist_y, hist_x):
     return S, biased_buckets
 
 
+def compute_outlier_buckets(outlier_values,
```
Comment (George3d6) [inline mindsdb/libs/phases/data_analyzer/data_analyzer.py:179]: Hard to compare side by side, but I assume this is just the old logic made into a function?

Or were there any changes made ?

File: test.py [hunk 4]
```diff
@@ -5,22 +5,31 @@
...
-
 lightwood.config.config.CONFIG.HELPER_MIXERS = False
 mdb = Predictor(name='test_predictor')
-#'rental_price',
-#mdb.learn(to_predict=['neighborhood','rental_price'],from_data="https://mindsdb-example-data.s3.eu-west-2.amazonaws.com/home_rentals.csv",use_gpu=False,stop_training_in_x_seconds=1000, backend='lightwood', unstable_parameters_dict={'use_selfaware_model':True})
-
-p = mdb.predict(when={'number_of_rooms': 3, 'number_of_bathrooms': 2, 'sqft':2411, 'initial_price':3000}, run_confidence_variation_analysis=True, use_gpu=True)
+# 'rental_price',
+mdb.learn(to_predict=['neighborhood', 'rental_price'],
+          from_data="https://mindsdb-example-data.s3.eu-west-2.amazonaws.com/home_rentals.csv",
```
Comment (George3d6) [inline test.py:12]: This seems to be an indentation issue

File: test.py [hunk 5]
```diff
@@ -5,22 +5,31 @@
...
+p_arr = mdb.predict(
+    when_data='https://mindsdb-example-data.s3.eu-west-2.amazonaws.com/home_rentals.csv',
+    use_gpu=False)
 print(p_arr[0].explanation)
 
 for p in p_arr:
     e = p.explanation
 
-p = mdb.predict(when={'number_of_rooms': 3, 'number_of_bathrooms': 2, 'neighborhood': 'south_side', 'sqft':2411}, run_confidence_variation_analysis=True, use_gpu=True)
+p = mdb.predict(when={'number_of_rooms': 3, 'number_of_bathrooms': 2,
```
Comment (George3d6) [inline test.py:30]: Why is there an empty line here? Or is this just a miss-fire of the github code compare ?

File: mindsdb/libs/helpers/general_helpers.py (hunk 2 above)
Comment (btseytlin) [reply mindsdb/libs/helpers/general_helpers.py:136]: Its broken. `col_name` is not defined in this function

File: mindsdb/libs/phases/data_analyzer/data_analyzer.py (hunk 3 above)
Comment (btseytlin) [reply mindsdb/libs/phases/data_analyzer/data_analyzer.py:179]: Yes, old logic made into a function

File: test.py (hunk 4 above)
Comment (btseytlin) [reply test.py:12]: There is no indentation issue, it runs. Though I accidentally committed this, I used this to test compiling mindsdb and it slipped though. Can revert if neened

File: test.py (hunk 5 above)
Comment (btseytlin) [reply test.py:30]: Definitely a miss-fire, in reality there is a new line, but no empty line

**Comment by George3d6** [comment]
Once the last issue I mentioned is fixed + the version is bumped feel free to merge this @btseytlin

File: mindsdb/libs/helpers/general_helpers.py (hunk 2 above)
Comment (btseytlin) [reply mindsdb/libs/helpers/general_helpers.py:136]: On another thought, I will uncomment it and make an issue about it

Comment (btseytlin) [reply mindsdb/libs/helpers/general_helpers.py]: https://github.com/mindsdb/mindsdb/issues/485

**Pull request merged**